pkg/virt-controller/watch/migration.go

```diff
@@ -248,41 +247,33 @@ func ensureSelectorLabelPresent(migration *virtv1.VirtualMachineInstanceMigratio
 }
 
 func (c *MigrationController) patchVMI(origVMI, newVMI *virtv1.VirtualMachineInstance) error {
-	var ops []string
+	fmt.Printf("XXX patchVMI\n")
+	patches := patch.NewPatchSet()
```

Comment:
Possibly rename / create a new package called `patchset`? IMO `patchset.New()` reads better than `patch.NewPatchSet()`.

Reply:
Actually, I'd like to replace everything with the patchset interface and remove completely also the `GenerateTestReplacePatch` and the types. About the naming, when I have refactor the code, it should be easy to rename the patch.go and the package to patchset